kernel/kernel-idgenerator-service/src/main/java/io/mosip/kernel/vidgenerator/constant/VIDHealthConstants.java

```diff
@@ -1,73 +0,0 @@
-package io.mosip.kernel.vidgenerator.constant;
-
-public class VIDHealthConstants {
```

Comment:
I am not sure if we are ok to remove files like this. Since we are already in LTS.

Reply:
Lets discuss this in the engineering review call

Reply:
Rolled back as discussed, added the files again.